libvisual/libvisual/private/lv_video_bmp.cpp

```diff
@@ -341,12 +343,14 @@ namespace LV {
           return nullptr;
       }
 
-      if (bi_compression > 3) {
+      if (bi_compression >= 3) {
```

Comment:
I'm not a big fan of approach "undocumented unnamed magic number". This seems to be 3 because `BI_BITFIELDS == 3` in Windows. If we cannot use that constant without including Windows headers, let's either (a) add a good comment or (b) make our own copy of that constant (or both)?